### PR TITLE
Add Stanislav Deviatov (me)

### PR DESCRIPTION
**Pull request by stn1slv**
I've added myself (Stanislav Deviatov)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1197,6 +1197,14 @@ https://twitter.com/simalexan
 Topics: Serverless, Offline Web, Chat Bots\
 [https://twitter.com/slobodan\_](https://twitter.com/slobodan_)
 
+### Novi Sad
+
+<img src="https://avatars.githubusercontent.com/u/20664330?v=4" height="70px" width="70px" align="left" alt="" />
+
+**Stanislav Deviatov**\
+Topics: Kafka, Mulesoft, API Management, Integration Platforms \
+[https://github.com/stn1slv](https://github.com/stn1slv)
```
Comment (karlhorky) [inline README.md]: we use Twitter or some other social media that allows contacting (GitHub has no messenger feature yet)

```suggestion
https://twitter.com/stn1slv
```

File: README.md [hunk 2]
```diff
@@ -1197,6 +1197,14 @@ https://twitter.com/simalexan
 Topics: Serverless, Offline Web, Chat Bots\
 [https://twitter.com/slobodan\_](https://twitter.com/slobodan_)
 
+### Novi Sad
+
+<img src="https://avatars.githubusercontent.com/u/20664330?v=4" height="70px" width="70px" align="left" alt="" />
```
Comment (karlhorky) [inline README.md]: we use the image that is associated with the link below, so switched this to a Twitter avatar too

```suggestion
<img src="https://res.cloudinary.com/dsscw65fc/image/twitter_name/stn1slv" height="70px" width="70px" align="left" alt="" />
```

File: README.md [hunk 3]
```diff
@@ -1197,6 +1197,14 @@ https://twitter.com/simalexan
...
+
+**Stanislav Deviatov**\
+Topics: Kafka, Mulesoft, API Management, Integration Platforms \
```
Comment (karlhorky) [inline README.md]:
```suggestion
Topics: Kafka, Mulesoft, API Management, Integration Platforms\
```

File: README.md (hunk 1 above)
Comment (stn1slv) [reply README.md]: @karlhorky, thanks for the review. 
Can I put a link to Telegram (as a messenger)? 
https://telegram.me/stn1slv

Comment (karlhorky) [reply README.md]: let's do a social media platform with public profile pages which allows for messaging the person

some Twitter alternatives I would accept are things like:

- Mastadon
- Bluesky

Comment (stn1slv) [reply README.md]: Thanks for the clarifications. Twitter is ok for me.

**Review by karlhorky — APPROVED** [review]
LGTM, thanks!

**Pull request merged**